### DISCUSSION TITLE
Does iosdev.space support the Mastodon Server Covenant?

### DISCUSSION THREAD

**Discussion by chrisjf** [iOSDevSpace/iosdevspace#13] [Q&A] [answered]
Hello iOS Devs! I was looking on the About page https://iosdev.space/about and didn't find anything regarding the Mastodon Server Covenant https://joinmastodon.org/covenant Is that something iosdev.space would be open to supporting?

**Comment by heyjaywilson** [comment, marked as answer]
Hi! Yes, iosdev.space does adhere to the covenant! I need to add that we adhere to it. I haven't asked to submit to it though.

Also, the server is hosted with [masto.host](https://masto.host/) so most of these are a given.
